CHANGELOG.md

```diff
@@ -148,6 +148,8 @@ All notable, unreleased changes to this project will be documented in this file.
 - Allow impersonating user by an app/staff - #7754 by @korycins:
   - Add `customerId` to `checkoutCustomerAttach` mutation
   - Add new permision `IMPERSONATE_USER`
+- Fix GraphQL federation support. - #7771 by @rafalp
```

Comment:
This should go to the 3.1 changelog I guess? We only release it in 3.1 or port it to 3.0 as well?

Reply:
Agreed that these changes should be in 3.0 and 3.1.